tests/samplers_tests/tpe_tests/test_sampler.py

```diff
@@ -1,4 +1,13 @@
 import pytest
+import random
+from typing import Callable
+from typing import Dict
+from typing import Optional
+from typing import Union
+from unittest.mock import Mock
+from unittest.mock import patch
```

Comment:
pytest is not standard library.

Reply:
Thanks for checking the PR! Yes, pytest is not, but [unittest](https://docs.python.org/3/library/unittest.html) is. Related PR: #1121.

Reply:
Nah, this test imports pytest before standard libraries.
https://github.com/optuna/optuna/pull/1126/files#diff-9b7cf9235ed9634dee5bbc85e55f8bb6R1

Reply:
Oh, wow, I didn't notice the first line. Thank you for catching it.